qdev_wrappers/templates/initialisation_script.py

```diff
@@ -4,34 +4,46 @@
 import numpy as np
 import matplotlib as mpl
 import matplotlib.pyplot as plt
-mpl.rcParams['figure.subplot.bottom'] = 0.15
-mpl.rcParams['font.size'] = 10
-
-from qdev_wrappers.file_setup import CURRENT_EXPERIMENT, my_init
-from qdev_wrappers.configreader import Config
-from qdev_wrappers.show_num import show_num, show_meta
-from qdev_wrappers.sweep_functions import do1d, do2d, do0d
-from qdev_wrappers.transmon import *
-from chickpea import Segment, Waveform, Element, Sequence
 
-# import instrument drivers
+# import instrument drivers from qcodes
 from qcodes.instrument_drivers.agilent.Agilent_34400A import Agilent_34400A
 from qcodes.instrument_drivers.QDev.QDac import QDac
 from qcodes.instrument_drivers.oxford.mercuryiPS import MercuryiPS
 from qcodes.instrument_drivers.rohde_schwarz.SGS100A import RohdeSchwarz_SGS100A
 from qcodes.instrument_drivers.yokogawa.GS200 import GS200
-from qdev_wrappers.cQED.customised_instruments import SR830_qQED, Decadac_cQED, AWG5014_cQED, \
- VNA_cQED, ATS9360Controller_cQED, AlazarTech_ATS9360_cQED, GS200_cQED, Keithley_2600_cQED, SphereCor
 
+# import customised qdev instuments from qdev_wrappers
+from qdev_wrappers.customised_instruments import SR830_cQED, Decadac_cQED, \
+    AWG5014_cQED, ATS9360Controller_cQED, AlazarTech_ATS9360_cQED, \
+    GS200_cQED, Keithley_2600_cQED, SphereCor
+
+# import locally customised instruments from local_instruments
+from local_instruments import my_very_local_VNA
+
+# import wrappers
+from qdev_wrappers.file_setup import CURRENT_EXPERIMENT, my_init, close_station
+from qdev_wrappers.configreader import Config
+from qdev_wrappers.show_num import show_num, show_meta
+from qdev_wrappers.sweep_functions import do1d, do2d, do0d
+from qdev_wrappers.transmon import *
+from chickpea import Segment, Waveform, Element, Sequence
```

Comment:
Should this be part of the template? I dont think people by default installs chickpea.

Reply:
sure, I can remove it.